yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -171,6 +175,12 @@ func (conv *DebeziumValueConverter) convertMap(tableName string, m map[string]*s
 		if err != nil {
 			return fmt.Errorf("fetch column schema: %w", err)
 		}
+		if !checkSourceExporter(exportSourceType) && strings.Contains(strings.ToLower(colType), "interval") {
```

Comment:
Use an equality check on `colType` instead of `strings.Contains()`.

Reply:
also checkout `strings.EqualFold` if that can be useful here.
https://pkg.go.dev/strings#EqualFold